pkgs/applications/science/math/gap/default.nix

```diff
@@ -61,13 +63,23 @@ in
 stdenv.mkDerivation rec {
   pname = "gap";
   # https://www.gap-system.org/Releases/
-  version = "4.11.1";
+  version = "4.12.0";
 
   src = fetchurl {
     url = "https://github.com/gap-system/gap/releases/download/v${version}/gap-${version}.tar.gz";
-    sha256 = "sha256-ZjXF2n2CdV+DOUhrnKwzdm9YcS8pfoI0+6QIGJAuowQ=";
+    sha256 = "sha256-YAXQwDFv6IA9hGgfML5k6s6DqVQvl19cs97frdQ5GEk=";
   };
 
+  postPatch = ''
+    # https://github.com/gap-system/gap/issues/5015
+    # TODO: remove after updating
+    sed -i 's|body = fail|body = false|' pkg/atlasrep/gap/access.gi
```

Comment:
This should not be necessary with 4.12.1, to be released during the GAP Days (October 17-21)